conf/scripts/init.sc

```diff
@@ -24,7 +27,7 @@ import org.apache.spark.rdd._
     val envJars = sys.env.get("ADD_JARS")
     val propJars = sys.props.get("spark.jars").flatMap { p => if (p == "") None else Some(p) }
     val jars = List(propJars, envJars).collect{case Some(j) => j}.mkString(",")
-    notebook.Utils.resolveURIs(jars).split(",").filter(_.nonEmpty)
+    resolveURIs(jars).split(",").filter(_.nonEmpty)
```

Comment:
yes, the thing is about initialization order. so it's impossible to initialise/compute value of `val trollMe`at initialization time (as vals are initialized in order).

making it `def or lazy` would still work...